spec/models/user_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe User do
+  it { is_expected.to validate_presence_of :username }
+
+  describe '#games' do
+    it 'includes games played as white' do
+      game = create(:game)
+
+      expect(game.white_player.games).to include game
```

Comment:
This totally works. I'd suggest changing the data setup a bit so that it's easier to follow.

```
let(:user) { create :user }

describe '#games' do
  it 'includes games played as white' do
    game_as_white_player = create :game, white_player: user

    expect(user.games).to include game_as_white_player
  end

  it 'includes games played as black' do
    game_as_black_player = create :game, black_player: user

    expect(user.games).to include game_as_black_player
  end
end
```